fastlane/lib/fastlane/actions/github_api.rb

```diff
@@ -16,7 +16,8 @@ def run(params)
           headers = construct_headers(params[:api_token], params[:headers])
           payload = construct_body(params[:body], params[:raw_body])
           error_handlers = params[:error_handlers] || {}
-          secure = params[:secure] || true
+          secure = params[:secure]
+          secure = true if secure.nil?
```

Comment:
We can remove this line (the one with the `if secure.nil?`) completely 💣 

Can you also add in `type: Boolean` into the `FastlaneCore::ConfigItem` for `:secure`? Like the following?
```rb
FastlaneCore::ConfigItem.new(key: :secure,
  env_name: "FL_GITHUB_API_SECURE",
  description: "Optionally disable secure requests (ssl_verify_peer)",
  is_string: false,
  type: Boolean,
  default_value: true,
  optional: true)
```

This provides a more secure default value and will make this work with `Fastlane.swift` 😊